sap-successfactors/4.0/modules/ROOT/pages/sap-successfactors-connector-reference.adoc

```diff
@@ -27,6 +27,10 @@ Anypoint Connector for SAP SuccessFactors (SuccessFactors Connector) provides fu
 [[config_basic]]
 == Basic Authentication 
 
+=== Note
+
+Basic Authentication requires Minimum Mule EE Version 4.2.0.
```

Comment:
Mule 4.2.0 and later is required for Basic authentication.

Reply:
This is more accurate